### ISSUE TITLE
[Idea] Change Raw Script into real raw script

### ISSUE THREAD

**Issue by ixje** [CityOfZion/neo-3-preview#80]
The transaction's page has a raw script section as shown below
![image](https://user-images.githubusercontent.com/6625537/81911201-495eb800-95cd-11ea-9de3-eedcaabacb82.png)

The data displayed is not actually "raw", it is still base64 encoded. My suggestion would be to base64 decode it and put the real raw script bytes in there.

**Comment by ixje** [comment]
the same suggestion also applies to notifications. e.g.
https://neo3-preview.com/transaction/0x019befa728a371a08fb0f78330b3c697ebc9d8bbf252bd173b17cc2bd6e97e9c

![image](https://user-images.githubusercontent.com/6625537/82867251-c9b8dd80-9f2a-11ea-9b99-7889b3413d5c.png)

The "HEXSTRING" version is still base64 encoded instead of the "real" hex string. If you toggle the format to "STRING" that does first decode it.

**Comment by lllwvlvwlll** [comment]
I agree that we probably want to display the values in the expected (decoded) format.  I could see scenarios where the notification has a byte[] publicKey for example.
